### PR TITLE
Correct naming for TPR creation

### PR DESCRIPTION
**Pull request by simonleung8**
fixes issue #428

`func TPRName()` comes from #441 

cc/ @arschles

### CODE REVIEW

**Comment by simonleung8** [comment]
CLA signed

**Comment by simonleung8** [comment]
I signed it!

File: pkg/storage/tpr/resources.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ var serviceClassTPR = v1beta1.ThirdPartyResource{
 	// ServiceClass is the kind, but TPRName converts it to 'serviceclass'. For now, just hard-code
 	// it here
 	ObjectMeta: v1.ObjectMeta{
-		Name: withGroupName("ServiceClass"),
+		Name: withGroupName(ServiceClassKind.TPRName()),
```
Comment (arschles) [inline pkg/storage/tpr/resources.go]: @simonleung8 can you change all the other resources to use `.TPRName()` too?

Comment (simonleung8) [inline pkg/storage/tpr/resources.go]: The other 3 resources are already using `.TPRName()` :)
https://github.com/kubernetes-incubator/service-catalog/blame/master/pkg/storage/tpr/resources.go#L98

**Comment by duglin** [comment]
CLA is now fixed

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
Added LGTM1 for @arschles

**Comment by arschles** [comment]
@simonleung8 CI is failing on a [vet error](https://travis-ci.org/kubernetes-incubator/service-catalog/builds/206605497#L497). Can you fix and ping me when done? I'll merge at that point

**Comment by duglin** [comment]
I've been trying to see those travis logs all morning w/o luck - that link you posted only worked for me once, then fails over and over.  Known travis issue....

**Comment by arschles** [comment]
@duglin don't know why but I could see it. Anyway, the issue is that the boilerplate header is missing from the new `pkg/storage/tpr/resources_test.go` file. @simonleung8 if you add that, hopefully we can get CI passing even though viewing the logs looks to be problematic.

**Comment by duglin** [comment]
@simonleung8 you may want to rebase on master too since if you don't your `make verify` might see other issues that are now fixed.

**Comment by pmorie** [comment]
@duglin @arschles I wasn't able to see them until like 2 am this morning, but I can see them now.

**Comment by pmorie** [comment]
LGTM, thanks for the fix.  Will merge on green ci

**Comment by arschles** [comment]
merging now

**Pull request merged**